modules/platforms/dotnet/Apache.Ignite.Core/Cluster/IClusterGroup.cs

```diff
@@ -259,5 +259,12 @@ public interface IClusterGroup
         /// <param name="enabled">Enabled flag</param>
         [SuppressMessage("Microsoft.Design", "CA1024:UsePropertiesWhereAppropriate", Justification = "Semantics.")]
         void EnableStatistics(IEnumerable<string> cacheNames, bool enabled);
+
+        /// <summary>
+        /// Clears statistics for caches cluster wide.
+        /// </summary>
+        /// <param name="caches">Collection of cache names.</param>
+        [SuppressMessage("Microsoft.Design", "CA1024:UsePropertiesWhereAppropriate", Justification = "Semantics.")]
```

Comment:
Do we really need this suppression? Is there a warning? Same for the method above.